lib/libzpool/taskq.c

```diff
@@ -353,11 +354,15 @@ system_taskq_init(void)
 {
 	system_taskq = taskq_create("system_taskq", 64, maxclsyspri, 4, 512,
 	    TASKQ_DYNAMIC | TASKQ_PREPOPULATE);
+	system_delay_taskq = taskq_create("system_taskq", 4, maxclsyspri, 4,
```

Comment:
It is a minor point, but I don't like duplicating the name "system_taskq". Lets call it something else. Maybe "zfs_zvol_prefetch_taskq".